TerminalDocs/customize-settings/actions.md

```diff
@@ -882,43 +946,20 @@ You can disable read-only mode on a pane. This works similarly to toggling, howe
 > [!IMPORTANT]
 > This feature is only available in [Windows Terminal Preview](https://aka.ms/terminal-preview).
```

Comment:
Are you going to do a separate pass to remove the old preview callouts? Will we be confused if we try to do it after this PR lands?

Reply:
Done, in https://github.com/MicrosoftDocs/terminal/pull/713